lib/tropic/tparagraph.flow

```diff
@@ -60,21 +60,18 @@ TropicParagraph(elements : [Tropic], minWidth : double, maxWidth : double, style
 	TLines(t);
 }
 
-// Analogue of likely named function in paragraph.flow
-applyRelativeScaleT(tx: TText) -> TText {
-	TText(tx.text, map(tx.style, \s -> {
-		switch (s) {
-			FontSize(v): {
-				// println(formatHex(getCharCodeAt(tx.text, 0))+" "+d2s(getCharRelativeScale(getCharCodeAt(tx.text, 0))));
-				FontSize(v * getCharRelativeScale(getCharCodeAt(tx.text, strlen(tx.text)-1)));
-			}
-			default: s;
+overrideTCharacterStylePerAlphabet(text: string, styles: [TCharacterStyle]) -> [TCharacterStyle] {
+	classified = fold(styles, Pair([], []), \acc, style -> {
```

Comment:
it's better to call filter 2 times than arrayPush in a loop

Reply:
Done